src/main/java/ApplyToGroup/ApplyToGroupRequestModel.java

```diff
@@ -0,0 +1,20 @@
+package ApplyToGroup;
+
+import Entities.Group;
+import Entities.User;
+
+import java.util.HashMap;
+
+public class ApplyToGroupRequestModel {
+    private String groupName;
+
+    public ApplyToGroupRequestModel(String groupName) {
+        this.groupName = groupName;
+    }
+
+    public String getGroupName() {
+        return groupName;
+    }
+
```

Comment:
This request model is looking really similar to mine. Maybe we should consider sharing, or having an interface that we can share? Not really sure because we have seperate use cases which should stay seperated for good organization but we can talk about that

Reply:
Yeah maybe it would be good to keep the request models seperate, even if they have similar parameters. I think the majority of the use cases have similar parameters but the response model for each use case should be totally isolated to that specific use case only.

Reply:
Oh, also. We've implemented the current user class, so we're passing around the current user username as well.

Reply:
Hello! I think that we should also keep the request models separate. If we can figure out a way that does not impact the organization negatively, we can discuss it before Phase 5.